test/UnitTests/Microsoft.OData.Core.Tests/UriParser/Binders/FunctionCallBinderTests.cs

```diff
@@ -1362,7 +1362,7 @@ public void GetUriFunction_OnlyCustomFunctionsExist()
             finally

```

Comment:
I noticed there are a lot of try{}catch. Since you have thread safe, is it time to remove all of them?